pixeltable/functions/uuid.py

```diff
@@ -0,0 +1,32 @@
+"""
+Pixeltable UDFs for `UUIDType`.
+"""
+
+import uuid
+
+import sqlalchemy as sql
+
+import pixeltable as pxt
+from pixeltable.utils.code import local_public_names
+
+
+@pxt.udf
+def make_uuid() -> uuid.UUID:
```

Comment:
Just call it uuid4()

We should maybe support at least some of the others as well? Not everyone prefers uuid4(). I don't know how those would map onto postgres.